tests/Unit/HtmlProcessor/HtmlPrunerTest.php

```diff
@@ -368,7 +383,7 @@ public function removeRedundantClassesNotRemovesClassesToKeep(string $html, arra
         $result = $subject->renderBodyContent();
         foreach ($classesToKeep as $class) {
             $expectedInstanceCount = \substr_count($html, $class);
-            self::assertSubstringCount($expectedInstanceCount, $result, $class);
+            $this->assertSubstringCount($expectedInstanceCount, $result, $class);
```

Comment:
Again, if I'm not mistaken, this is a `static` method and should not be called via `$this`.  Same comment applies: is there an issue with Psalm?

Reply:
yup, NonStaticSelfCall  https://github.com/MyIntervals/emogrifier/commit/015728397f9dd995afbbb20ee1bd6483967cfabc#diff-db03589574d7c5d074966c97658929d9

Reply:
This may be because of a Psalm bug (see #792), but may perhaps be because you've also changed the `assertSubstringCount` method signature to be non-`static` (https://github.com/SignpostMarv/emogrifier/blob/psalm/tests/Unit/HtmlProcessor/HtmlPrunerTest.php#L632)?

Reply:
reverted in rebase referenced https://github.com/MyIntervals/emogrifier/pull/778#issuecomment-538681924